tests/utils/test_search_utils.py

```diff
@@ -328,6 +328,14 @@ def test_invalid_order_by_search_runs(order_by, error_message):
     assert error_message in e.value.message
 
 
+def test_space_order_by_search_runs():
+    identifier_type, identifier_name, ascending = SearchUtils.parse_order_by_for_search_runs(
+        "metrics.`Mean Square Error`")
+    assert identifier_type == "metric"
+    assert identifier_name == "Mean Square Error"
+    assert ascending
```

Comment:
```suggestion
@pytest.mark.parametrize("order_by, ascending_expected", [
    ("metrics.`Mean Square Error`", True),
    ("metrics.`Mean Square Error` ASC", True),
    ("metrics.`Mean Square Error` DESC", False),
])
def test_space_order_by_search_runs(order_by, ascending_expected):
    identifier_type, identifier_name, ascending = SearchUtils.parse_order_by_for_search_runs(
        order_by)
    assert identifier_type == "metric"
    assert identifier_name == "Mean Square Error"
    assert ascending == ascending_expected
```
@jdlesage Sorry for the delay. Can we add two more test cases to ensure that order_by query with `ASC|DESC` annotation works too?

Reply:
Done

Reply:
@jdlesage Thanks!